lib/semian/atomic_integer.rb

```diff
@@ -0,0 +1,28 @@
+module Semian
+  class AtomicInteger #:nodoc:
+    attr_accessor :value
+
+    def initialize(_name, _permissions)
```

Comment:
Yes. I don't have a good idea of what this will look like, but the in-memory Ruby adapter can't take permissions just because another adapter happens to need it.